eq-author/src/App/publish/PublishPage.js

```diff
@@ -113,10 +113,21 @@ const PublishPage = ({ match, history }) => {
   };
 
   const publishStatus = questionnaire && questionnaire.publishStatus;
+
+  const canPublish = () => {
+    if (!questionnaire) {
+      return true;
```

Comment:
Should we allow publishing if the questionnaire is falsy?

Reply:
questionnaire should not stay falsy. It was just an alternative to using the loading page.

Reply:
But why not return false so we stop them from seeing the publish page while the questionnaire is loading?

Reply:
This is for redirecting. Could return null or something on the page to prevent them from seeing the publish page while it is waiting for the questionnaire object to become available. But I dont mind showing the publish page instantly, after all it is the page you are going to. Redirect will kick in if they cant publish.